src/rgw/rgw_rest_s3.cc

```diff
@@ -1688,6 +1688,17 @@ if(!continuation_token_exist) {
 return 0;
 }
 
+void RGWListBucket_ObjStore_S3::dump_objname(const char* key, const std::string& val) {
```

Comment:
could we make dump_objname a static inline function in this file, rather than a new member function on the S3 bucket op?  (more like dump_owner, though that's not quite ideal either...)

Reply:
i mean, what's the advantage of making it static?

Reply:
guess it is intended to be called by more ops, made it static.

Reply:
yes, it's in a free function, with inline, it allows the compiler to inline it into the current translation unit